<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>551</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton1">
      <property name="text">
       <string>네모로직 만들기(그림)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton2">
      <property name="text">
       <string>타임어택</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton3">
      <property name="text">
       <string>네모로직 문제</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton4">
      <property name="text">
       <string>자동 풀이</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton5">
      <property name="text">
       <string>종료하기</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>720</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Square Logic</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
